torch/_subclasses/fake_tensor.py

```diff
@@ -655,6 +662,13 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
             else:
                 return args[0].fake_device
 
+        # Some attribute queries that can be serviced directly
+        # See Note [is_coalesced is dispatched]
+        if func in [torch.ops.aten.is_coalesced.default]:
```

Comment:
reason not to register this with `register_op_impl` ?

Reply:
It needs to happen early. In particular, it needs to happen before we check for non-fake, as the triggering case was we are in fake tensor mode querying if a non fake tensor is coalesced.